cli2telegram/cli.py

```diff
@@ -81,19 +82,24 @@ def cli(bot_token: str or None, chat_id: str or None, lines: Tuple[str], daemon:
         return
 
     LOGGER.debug("Processing message...")
-    prepared_message = prepare_code_message(lines)
+    messages = split_message(lines)
 
-    if len(prepared_message) <= 0:
+    if len(messages) <= 0:
```

Comment:
This doesn't do the same thing now.
The check is supposed to prevent cli2telegram from sending empty messages, because the telegram API will reject those. So to have the same effect as before, you would need to check for each message in messages, if their `length <= 0` (f.ex. in the loop below).

Reply:
Since multiple messages a created only if the text message is longer than the limit, it should imply that each message is not empty. The only possible scenario for an empty message is if the text message is empty, but there is already a check on the number of lines before this one that should cover that.

Reply:
Ah right, I missed that part because it was collapsed by github.

Reply:
Should I remove that?